packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -472,6 +485,14 @@ export default {
     }
   },
   methods: {
+    ...mapActions('Files/sidebar', ['openWithPanel']),
+    openSharingSidebar(file) {
+      if (file.shareTypes?.includes(ShareTypes.link.value)) {
```

Comment:
I added this myself, but the shareTypes field is not available when the function is called. Need to check on monday.

Reply:
seems to work fine

Reply:
for me the `shareTypes` field is not available in the moment the button is clicked

Reply:
@kulmann fixed it by using share?.shareType